nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/resources/CollectionLeafResource.java

```diff
@@ -355,8 +356,19 @@ private String getErrorMessage(Map<String, String> data, ActionReport ar) {
                 options.append(sep).append(removeVersioning ? new JvmOption(key).option : key);
 
                 String value = entry.getValue();
-                if (value != null && !value.isEmpty() || key != null && key.startsWith("-D")) {
-                    options.append("=").append(entry.getValue());
+                
+                if (key != null && !key.trim().isEmpty() && key.startsWith("-D")) {    
+                    if (value == null || value.equals("null")) {
```

Comment:
Could the text "null" ever be a valid value?

Reply:
Upon further thought, I think it should be accepted, "null" is a perfectly valid Gaelic name.